Make stage - ruby dependency error

**This is a...** 
- [ ] Feature Request
- [x] Bug Report

**Problem:**

```
$ git clone git@github.com:kubernetes/website.git
$ cd website
$ make stage
docker run -ti --rm -v "/Users/westcos/code/kubernetes/website":/k8sdocs -p 4000:4000 gcr.io/google-samples/k8sdocs:1.1
Unable to find image 'gcr.io/google-samples/k8sdocs:1.1' locally
1.1: Pulling from google-samples/k8sdocs
53ebc9bfbcc0: Pull complete
5383cbbe6a7c: Pull complete
a71e7f099d62: Pull complete
ad65daa78430: Pull complete
b61497b95f74: Pull complete
Digest: sha256:6593f2261ddb81a8e0f9e597dfb7b48c8cf8a5cc118cf029086480ed15181567
Status: Downloaded newer image for gcr.io/google-samples/k8sdocs:1.1
+ bundle
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and installing your bundle as root will break this
application for all non-root users on this machine.
Warning: the running version of Bundler (1.14.6) is older than the version that created the lockfile (1.16.1). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Fetching gem metadata from https://rubygems.org/.........
Fetching version metadata from https://rubygems.org/..
Fetching dependency metadata from https://rubygems.org/.
ruby_dep-1.5.0 requires ruby version >= 2.2.5, which is incompatible with the current version, ruby 2.2.4p230
make: *** [stage] Error 5
```

**Proposed Solution:**
Changing the base image in `staging-container/Dockerfile` to alpine:3.7 worked for me locally. Happy to raise a PR if the maintainers agree with doing this.

--- a/staging-container/Dockerfile
+++ b/staging-container/Dockerfile
@@ -1,4 +1,4 @@
-FROM alpine:3.3
+FROM alpine:3.7
 
 RUN apk add --no-cache \
 	build-base \

